crates/houston/src/error.rs

```diff
@@ -0,0 +1,31 @@
+use thiserror::Error;
+
+use std::io;
+
+/// HoustonProblem is the type of Error that occured.
+#[derive(Error, Debug)]
+pub enum HoustonProblem {
+    /// ConfigDirNotFound occurs when the default OS config can't be found.
+    #[error("Could not determine default OS config directory.")]
```

Comment:
Is there a way to make these messages more actionable?

Reply:
the actionable error messages will have to come directly from the CLI implementation, I left them out of this crate so as to be less opinionated about the error messages displayed to the user, the CLI as the consumer should have control over those messages. This PR does not include those changes however.